src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -249,7 +249,8 @@ public static void setUpProxy() throws Exception {
       } else {
         clientPrincipal = "root";
         tokenClass = PasswordToken.class.getName();
-        sharedSecret = SharedMiniClusterBase.getRootPassword();
+        sharedSecret = "sharedSecret";
```

Comment:
I would make the value different from the key that's used in the config file... just to make sure that it's detectable if we accidentally read the key instead of the value or vice versa.